src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/goal/service/ActionPlanService.kt

```diff
@@ -20,7 +23,9 @@ class ActionPlanService(
    * Retrieves a Prisoner's [ActionPlan] based on their prison number.
    * Throws [ActionPlanNotFoundException] if the [ActionPlan] cannot be found.
    */
-  fun getActionPlan(prisonNumber: String): ActionPlan =
-    persistenceAdapter.getActionPlan(prisonNumber)
+  fun getActionPlan(prisonNumber: String): ActionPlan {
+    log.debug { "Retrieving Action Plan for prisoner [$prisonNumber]" }
```

Comment:
Minor - should this be `log.info` on the basis that the other 2 are?
Or is there some rationale why this is `debug` ?

Reply:
There is a rational :) I think creating an ActionPlan/Goal is useful information that various people (whoever they may be) might be interested in, especially in the absence of reporting.

Reply:
Ah, that makes sense 👍